### ISSUE TITLE
Implement virtual morse code input per specification

### ISSUE THREAD

**Issue by TechSolomon** [andrewsng/assistive-technology#26]
```cs
lowAudio(35) = "zero"
lowAudio(36) = "period"
lowAudio(37) = "comma"
lowAudio(38) = "question mark"
lowAudio(39) = "forward slash"
lowAudio(40) = "exclamation point"
lowAudio(41) = "apostrophe"
lowAudio(42) = "semicolon"
lowAudio(43) = "colon"
lowAudio(44) = "dash"
lowAudio(45) = "underline"
lowAudio(46) = "left parenthesis"
lowAudio(47) = "right parenthesis"
lowChar(35) = "0"
lowChar(36) = "."
lowChar(37) = ","
lowChar(38) = "?"
lowChar(39) = "/"
lowChar(40) = "!"
lowChar(41) = "'"
lowChar(42) = ";"
lowChar(43) = ":"
lowChar(44) = "-"
lowChar(45) = "_"
lowChar(46) = "("
lowChar(47) = ")"
code(0) = ".-"       'a'
code(1) = "-..."     'b'
code(2) = "-.-."     'c'
code(3) = "-.."      'd'
code(4) = "."        'e'
code(5) = "..-."     'f'
code(6) = "--."      'g'
code(7) = "...."     'h'
code(8) = ".."       'i'
code(9) = ".---"     'j'
code(10) = "-.-"     'k'
code(11) = ".-.."    'l'
code(12) = "--"      'm'
code(13) = "-."      'n'
code(14) = "---"     'o'
code(15) = ".--."    'p'
code(16) = "--.-"    'q'
code(17) = ".-."     'r'
code(18) = "..."     's'
code(19) = "-"       't'
code(20) = "..-"     'u'
code(21) = "...-"    'v'
code(22) = ".--"     'w'
code(23) = "-..-"    'x'
code(24) = "-.--"    'y'
code(25) = "--.."    'z'
code(26) = ".----"   '1'
code(27) = "..---"   '2'
code(28) = "...--"   '3'
code(29) = "....-"   '4'
code(30) = "....."   '5'
code(31) = "-...."   '6'
code(32) = "--..."   '7'
code(33) = "---.."   '8'
code(34) = "----."   '9'
code(35) = "-----"   '0'
code(36) = ".-.-.-"  '.'
code(37) = "--..--"  ','
code(38) = "..--.."  '?'
code(39) = "-..-."   '/'
code(40) = "......." '!'
code(42) = "-.-.-"   ';'
code(43) = "---..."  ':'
code(44) = "-....-"  '-'
code(45) = "..--.-"  '_'
code(46) = "-.--.-"  '('
code(47) = "-.--.."  ')'
code(65) = "..--"    '@'
```

**Comment by TechSolomon** [comment]
See also (escape characters): `code(41) = ".----." as '`

**Comment by TechSolomon** [comment]
Also consider how the last letter is stored in memory (e.g. second L in the "Hello" video).

Code reference area per implementation guides data structure decision: https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2?view=net-8.0

**Comment by TechSolomon** [comment]
Closed per 01cb9fb59ea13df77374f8bce0f893910ef21a7d.
